packages/pwa-kit-react-sdk/package.json

```diff
@@ -26,7 +26,7 @@
     "build:watch": "watch 'npm run build' src/",
     "lint": "npm run lint:js",
     "lint:fix": "npm run lint:js -- --fix",
-    "lint:js": "internal-lib-build lint \"**/*.{js,jsx}\"",
+    "lint:js": "cross-env ../../node_modules/.bin/eslint \"**/*.{js,jsx}\"",
```

Comment:
Instead of assuming that we'll find an `eslint` binary here `../../node_modules/.bin/eslint` looks like we should add eslint as a `devDependency` and use it in a script like `"eslint": "eslint"`as a shortcut for the local `node_modules/.bin/eslint`.